examples/c/chain_model/chain_model.h

```diff
@@ -7,6 +7,24 @@
 extern "C" {
 #endif
 
+#define X0_NM2_FILE "/home/bnovoselnik/Software/acados/examples/c/chain_model/x0_nm2.txt"
```

Comment:
This file got changed my cmake... We should find a better way of doing that, everyone added it to a commit at least once :p

Reply:
I agree, it's really annoying :)